doc/sphinxman/source/occ.rst

```diff
@@ -425,4 +448,10 @@ Starting in v1.4, MP2.5 and MP3 default to the density-fit algorithm. Set |globa
     +                         +--------------------------------------------------------------+----------------------+----------------------+---------------------------+
     |                         | Cholesky-Decomposed Lambda-CCSD(T)                           | RHF                  | ---                  | |globals__cc_type| CD     |
     +-------------------------+--------------------------------------------------------------+----------------------+----------------------+---------------------------+
+    | remp2                   | 2nd order REMP hybrid perturbation theory                    | RHF/UHF              | ---                  | |globals__cc_type| CONV   |
```

Comment:
_Technically_ we don't have density-fitted yet. If we're on track to get that in by 1.7, I leave it up to you whether you want this part of the docs to be accurate in the meantime.